bin/alluxio-start.sh

```diff
@@ -12,18 +12,19 @@ BIN=$(cd "$( dirname "$0" )"; pwd)
 USAGE="Usage: alluxio-start.sh [-hNw] WHAT [MOPT] [-f]
 Where WHAT is one of:
   all MOPT\t\tStart master and all workers.
```

Comment:
is MOPT optional for `all`? if not, could we follow similar convention as for `local`?